### PR TITLE
Feature/initial login flow

### PR DESCRIPTION
**Pull request by poocart**
## Description

- Added `privy` and login flows with App loading (splashscreen)  component.
- Added initial bottom menu bar.
- Replaced default `create-react-app` test suite with generic `jest`.
- Added/reorganized related components.

## How Has This Been Tested?

- Locally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -25,10 +27,30 @@ const App = () => {
     <ThemeProvider theme={defaultTheme}>
       <GlobalStyle />
       <LanguageProvider>
-        <RouterProvider router={navigation} />
+        <PrivyProvider
+          appId={process.env.REACT_APP_PRIVY_APP_ID as string}
+          config={{ appearance: { theme: 'light' } }}
```
Comment (IAmKio) [inline src/App.tsx:32]: Note: we need to add a light / dark mode switcher later

File: src/navigation/index.tsx [hunk 2]
```diff
@@ -1,11 +1,50 @@
...
+import { Route, Routes } from 'react-router-dom';
 
-import Home from '../pages/Home';
+// pages
+import Account from '../pages/Account';
+import Transfer from '../pages/Transfer';
+import History from '../pages/History';
+import NotFound from '../pages/NotFound';
+import Apps from '../pages/Apps';
+import { usePrivy } from '@privy-io/react-auth';
```
Comment (IAmKio) [inline src/navigation/index.tsx:10]: Should this be moved before / after pages as it's a library?

File: src/navigation/index.tsx [hunk 3]
```diff
@@ -1,11 +1,50 @@
...
+  home: '/',
+  transfer: '/transfer',
+  history: '/history',
+  apps: '/apps',
+}
+
+const Navigation = () => {
+  const { ready, authenticated } = usePrivy();
+
+  console.log('ready', ready)
```
Comment (IAmKio) [inline src/navigation/index.tsx:24]: That reminds me, i'll get Sentry set up

File: src/theme/index.ts [hunk 4]
```diff
@@ -19,9 +25,15 @@ export const defaultTheme: Theme = {
...
       body: '#6448b3',
+      bottomMenuItem: '#fff',
+      loadingLogo: '#fff',
+      buttonPrimary: '#c63bea',
     },
     text: {
       body: '#fff',
+      bottomMenuItem: '#000',
+      loadingLogo: '#997cfa',
+      buttonPrimary: '#fff',
```
Comment (IAmKio) [inline src/theme/index.ts:36]: Just a note for later to implement light / dark mode

**Review by IAmKio — CHANGES_REQUESTED** [review]
One small outstanding issue but the rest are just notes for myself for a future feature / PR - great work 🎉

File: src/theme/index.ts (hunk 4 above)
Comment (poocart) [reply src/theme/index.ts:36]: theming works oob so any other theme object passed would work 👍

File: src/navigation/index.tsx (hunk 2 above)
Comment (poocart) [reply src/navigation/index.tsx:10]: yes, actually I will go through each import and break in commented sections

File: src/theme/index.ts (hunk 4 above)
Comment (IAmKio) [reply src/theme/index.ts:36]: ty

File: src/navigation/index.tsx (hunk 3 above)
Comment (poocart) [reply src/navigation/index.tsx:24]: ah yes! or as we have Firebase we can also rely on Firebase Crashlytics?

**Review by IAmKio — APPROVED** [review]
LGTM 🚀

**Pull request merged**